golang/go.mod

```diff
@@ -1,25 +1,18 @@
 module github.com/summa-tx/relays/golang
 
-go 1.12
+go 1.15
 
 require (
 	github.com/cosmos/cosmos-sdk v0.37.11
 	github.com/gorilla/mux v1.7.0
-	github.com/mattn/go-isatty v0.0.10 // indirect
-	github.com/onsi/ginkgo v1.10.3 // indirect
-	github.com/onsi/gomega v1.7.1 // indirect
-	github.com/pelletier/go-toml v1.6.0 // indirect
-	github.com/spf13/afero v1.2.2 // indirect
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/viper v1.6.1
-	github.com/stretchr/testify v1.4.0
+	github.com/stretchr/testify v1.5.1
 	github.com/summa-tx/bitcoin-spv/golang v1.4.0
+	github.com/summa-tx/relays/proto v0.0.0-00010101000000-000000000000
 	github.com/tendermint/go-amino v0.15.0
 	github.com/tendermint/tendermint v0.32.10
 	github.com/tendermint/tm-db v0.2.0
-	golang.org/x/crypto v0.0.0-20190923035154-9ee001bba392 // indirect
-	golang.org/x/net v0.0.0-20190923162816-aa69164e4478 // indirect
-	golang.org/x/sys v0.0.0-20191128015809-6d18c012aee9 // indirect
-	golang.org/x/text v0.3.2 // indirect
-	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
 )
+
+replace github.com/summa-tx/relays/proto => ../proto
```

Comment:
Oh, that's how you get it to recognize the proto package!

Reply:
yehhhh go package stuff is a nightmare